### PR TITLE
feat/migrate embedders over

### PR DESCRIPTION
**Pull request by rbiseck3**
### Description
Move all embed code over to the new repo from the unstructured repo. As part of the migration, the implementation was also updated to remove any reference to the original repo and use generic dict type rather than `Element` when processing the content. 

Should be deployed in sync with this PR in the other repo to mark it deprecated there: https://github.com/Unstructured-IO/unstructured/pull/3614

### CODE REVIEW

File: unstructured_ingest/embed/voyageai.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    @requires_dependencies(
+        ["langchain", "langchain_voyageai"],
+        extras="embed-voyageai",
+    )
+    def get_client(self) -> "VoyageAIEmbeddings":
+        """Creates a Langchain VoyageAI python client to embed elements."""
+        from langchain_voyageai import VoyageAIEmbeddings
+
+        return VoyageAIEmbeddings(
+            voyage_api_key=self.api_key,
```
Comment (potter-potter) [inline unstructured_ingest/embed/voyageai.py:29]: does this need get_secret_key?

File: test_e2e/dest/chroma.sh [hunk 2]
```diff
@@ -22,8 +21,7 @@ source "$SCRIPT_DIR"/cleanup.sh
 
 function cleanup() {
   # Kill chroma background process
-  pgrep -f chroma-dest | xargs kill
-  cleanup_dir "$DESTINATION_PATH"
+  docker compose -f "$SCRIPT_DIR"/env_setup/chroma/docker-compose.yml down --remove-orphans -v
```
Comment (potter-potter) [inline test_e2e/dest/chroma.sh:24]: Thanks for switching this to docker...

**Comment by potter-potter** [comment]
If you could bring this test over please.
https://github.com/Unstructured-IO/unstructured/blob/main/test_unstructured_ingest/src/local-embed-mixedbreadai.sh

File: unstructured_ingest/embed/voyageai.py (hunk 1 above)
Comment (rbiseck3) [reply unstructured_ingest/embed/voyageai.py:29]: So the voyager embeddings is also using a secret string in a base model, so passing this straight through is just fine .

**Review by potter-potter — APPROVED** [review]
looks good

**Pull request merged**